manage/deploying/front-end/nginx.rst

```diff
@@ -121,14 +125,18 @@ If, and only if, you cannot use a platform install of nginx you may use the reci
 A buildout will download, install and configure nginx from scratch.
 The buildout file contains an nginx configuration which can use template variables from ``buildout.cfg`` itself.
 
-When you change the configuration of nginx in buildout you probably don't want to rerun the whole buildout, but only the nginx part of it::
+When you change the configuration of nginx in buildout, you probably don't want to rerun the whole buildout, but only the nginx part of it:
+
+.. code-block:: console
 
```

Comment:
we use shell -> .. code-block:: shell -> https://docs.plone.org/about/rst-styleguide.html